config/settings.yml

```diff
@@ -1,4 +1,28 @@
-plugins:
+just_chat:
   just_chat_enabled:
-    default: false
+    default: true
     client: true
+  discourse_narrative_bot_enabled:
+    default: false
+    hidden: true
```

Comment:
Does this override core's declaration of the site setting? I'm not sure and I don't think we should be redefining core site settings since it may lead to drifts in the future.